### ISSUE TITLE
Only uploads extensions.json, nothing else

### ISSUE THREAD

**Issue by mndscp** [nonoroazoro/vscode-syncing#87]
### Specifications
Version: 1.59.1 (user setup)
Commit: 3866c3553be8b268c8a7f8c0482c0c0177aa8bfa
Date: 2021-08-19T11:56:46.957Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Windows_NT x64 10.0.19041

### Expected Behavior
Upload all settings to GitHub Gist so the exact setup can be reproduced on another machine.

### Actual Behavior
Only uploads a single file "extensions.json" to the Gist. When downloading settings from another machine all the extensions get installed but everything else is missing. No color theme, no keybindings, and all that.

### Steps to Reproduce
Install this extension, upload settings, copy Gist ID to another machine, download settings there.

---

Maybe the synchonization thinks that I haven't updated any files except extensions? But as I am using a new Gist everything should be reset so everything will be uploaded. Or there could be an option to clear the current status or cache. Please let me know :)

**Comment by joe733** [comment]
Have tried with [PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)?

Because it works for me (two days ago):

![image](https://user-images.githubusercontent.com/38415384/140026893-c7e1606d-c177-4beb-a280-f0557d3642f1.png)

**Comment by nonoroazoro** [comment]
@mndscp If it's resolved, I'd close this issue~

**Comment by mndscp** [comment]
Hey guys, I am sorry for not responding in time.
Thanks for your suggestion @joe733

Last year I was on Windows and I thought I had found the problem, it being that it only uploads `extensions.json` when using the WSL on Windows compared to using the Windows-integrated version.
But I am now on Manjaro Linux, re-installed the extension today, created a PAT with `gist` checked, add when uploading with Syncing it once again only uploads a single `extensions.json` even though I have made changes to snippets and of course my complete `settings.json`.

I am now using VSCodium if that's any help.
Is there a "files that were changed" status that I can reset for Syncing? Otherwise I have no idea why this happens.
